pkg/apis/extensions/types.go

```diff
@@ -30,27 +30,26 @@ package extensions
 
 import (
 	"k8s.io/kubernetes/pkg/api"
-	"k8s.io/kubernetes/pkg/api/resource"
 	"k8s.io/kubernetes/pkg/api/unversioned"
 	"k8s.io/kubernetes/pkg/util"
 )
 
-// ScaleSpec describes the attributes a Scale subresource
+// ScaleSpec describes the attributes of a Scale subresource
 type ScaleSpec struct {
-	// Replicas is the number of desired replicas. More info: http://releases.k8s.io/HEAD/docs/user-guide/replication-controller.md#what-is-a-replication-controller"
+	// Replicas is the number of desired replicas. Setting this number will set the number of the desired replicas in the underlying replication controller/deployment.
```

Comment:
"Replicas is the desired number of instances for the scaled object."

I would remove the second sentence.

Replicas is the number of replicas is redundant ;-)